ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "data.xmldata"
-version = "1.3.2"
+version = "1.4.0"
```

Comment:
Is this a breaking change, cant we do a patch release for this?

Reply:
not a breaking change we can do a patch release